### PR TITLE
Stomp protocol added in order to notify brokers such as ActiveMQ

### PR DESCRIPTION
**Pull request by snuids**
A new type of alerter added in order to send alerts to brokers such as ActiveMQ or RabbitMQ.

Will probably add the AMQP protocols which is widely used with classic middleware.

Example of alert configuration:
# (Required)
# The alert is use when a match is found

alert:
- "stomp"
# (required, stomp specific)

stomp_hostname: 127.0.0.1
stomp_hostport: 61613
stomp_login: admin
stomp_password: admin
stomp_destination: /queue/ALERT3

### CODE REVIEW

File: elastalert/util.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from elasticsearch import RequestsHttpConnection
 from elasticsearch.client import Elasticsearch
 
-logging.basicConfig()
+logging.basicConfig(level=logging.INFO, filename='./alerter.log')
```
Comment (Qmando) [inline elastalert/util.py]: I think I would rather see file logging as a setting, not the default.

**Comment by snuids** [comment]
How do you want the documentation. Do you want me to prepare a text file compatible with the current doc ? (https://github.com/Yelp/elastalert/edit/master/docs/source/ruletypes.rst)

File: elastalert/util.py (hunk 1 above)
Comment (snuids) [inline elastalert/util.py]: I will remove that

**Comment by snuids** [comment]
Can I do something to close the pull ?

**Pull request merged**